web/Cargo.toml

```diff
@@ -9,7 +9,28 @@ crate-type = ["cdylib"]
 
 [dependencies]
 common = { path = "../common" }
+web-sys = "0.3"
+js-sys = "0.3"
+futures = "0.3"
+async-trait = "0.1"
 wasm-bindgen = "0.2"
 wasm-bindgen-futures = "0.4"
-async-trait = "0.1"
-js-sys = "0.3"
+console_log = "0.2"
+log = "0.4"
+tokio = "0.3"
```

Comment:
I think I may be missing something, but where is tokio used?

Reply:
Good catch, tried Tokio channel/sync stuff at some point, then forgot to remove it.